classes/class.ilTestCronConfigGUI.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+
+use iLUB\Plugins\TestCron\Helper\DIC;
+
+/**
+ * Class ilTestCronConfigGUI
+ *
+ *
+ * @package
+ */
+class ilTestCronConfigGUI extends ilPluginConfigGUI {
+
+	use DIC;
```

Comment:
grml, I hate this pattern, know I know why. Remove. For more questions, we can discuss this. Inject dic only through constructor (he most "outer" of your classes must use the "global" keyword).